TMDS_3/include/CGAL/IO/output_to_vtu.h

```diff
@@ -351,7 +351,17 @@ void output_to_vtu_with_attributes(std::ostream& os,
 
 } // namespace internal
 
-//public API
+//! \ingroup PkgTMDS3ExportFunctions
+//!
+//! \brief exports a tetrahedral mesh complex using the `UnstructuredGrid` XML format.
```

Comment:
Maybe add a link to https://kitware.github.io/vtk-examples/site/VTKFileFormats/#unstructuredgrid

Reply:
The other output functions add the link as a See Also.